examples/vercel-ai-chatbot/package.json

```diff
@@ -24,7 +24,7 @@
     "@vercel/analytics": "^1.1.2",
     "@vercel/kv": "^1.0.1",
     "@vercel/og": "^0.6.2",
-    "ai": "^2.2.31",
+    "ai": "^3.0.5",
```

Comment:
Will merge this and the package-lock changes in separately to focus this PR a bit more